canonical-websites/www.ubuntu.com
# add a redirect for /usn with no page

## Done

- add a redirect for /usn with no page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
  - https://www.ubuntu.com/usn and see that you go to https://usn.ubuntu.com/
- go to https://www.ubuntu.com/usn/usn-3643-2/ and see that you still go to https://usn.ubuntu.com/3643-2/

## Issue / Card

Fixes [RT #111049](https://rt.admin.canonical.com/Ticket/Display.html?id=111049)
